tests/requirements.txt

```diff
@@ -9,11 +9,13 @@ conda-build
 conda-forge::pre-commit
 conda-forge::pytest-split
 conda-forge::pytest-xprocess
+conda-forge::werkzeug >=2.2
 conda-forge::xdoctest
 conda-package-handling
 conda-verify
 filelock
 flake8
+conda-forge::flask >=2.2
```

Comment:
will be necessary for jlap tests

Reply:
Have we in between gotten these versions on defaults?

Reply:
No. Recent jinja2 and werkzeug are available though. https://anaconda.org/main/flask

Reply:
I may be able to defer this until the next branch.